style-rtl.css

```diff
@@ -813,6 +813,13 @@ textarea {
 	border-color: #dcd7ca;
 }
 
+code,
+input[type="url"],
+input[type="email"],
+input[type="tel"] { /* stylelint-disable-line */
+	direction: rtl;
```

Comment:
Is this supposed to say ltr?

Reply:
It's in the rtl stylesheet, so I assume that it's supposed to be rtl

Reply:
Yes, it should say `LTR` instead of `RTL`. I'll double-check how this could happen, @carolinan. Thanks for pointing it out! 🙌

Reply:
It seemed the combinations of build and lint statements fails here:

```
code,
input[type="url"],
input[type="email"],
input[type="tel"] {
	/*rtl:ignore*/ /* stylelint-disable-line */
	direction: ltr;
}
```

I fixed it by changing the part to:

```
code,
input[type="url"],
input[type="email"],
input[type="tel"] {

	/*rtl:ignore*/
	direction: ltr;
}
```

Reply:
@ntwb I'm sure you can assist me to solve the following mystery. First I tried

`/*rtl:ignore*/ /* stylelint-disable-line */`

which failed. Then I tried 

`/* rtl:ignore stylelint-disable-line */`

which also failed. Last but not least, I tried

```
/*rtl:ignore*/
/* stylelint-disable-line */
```

which, again, failed. I now ended up with the following code:

```
code,
input[type="url"],
input[type="email"],
input[type="tel"] {

	/*rtl:ignore*/
	direction: ltr;
}
```

That code is far from elegant and while it works, it does not look correct to me. Can you tell me how to avoid the empty line while still respect linting?